### ISSUE TITLE
NTR: Nasal biopsy: hamartomas (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9235]
New term request Nasal biopsy: hamartomas (elthat_110419020353):   UMLS: 5420040

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000078
- label: Nasal hamartoma
- parents: [Hamartoma (HP:0010566)](https://hpo.jax.org/app/browse/term/HP:0010566)
- pmid: 
- def: A hamartoma (tissue malformation consisting of an abnormal mixture of constitutive components) originating in the nose.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9235
- synonyms:
